salt/resource-manager/init.sls

```diff
@@ -92,3 +93,36 @@ resource-manager_spark_script_move:
     - makedirs: True
     - onlyif: 
       - test -x /usr/bin/spark-script-wrapper.sh
+
+{% for exec in flink_execs%}
```

Comment:
is there a better way to call the for loop which includes spark & flink execs in one loop and have a if/else to call it's respective wrapper.sh? As the same code is getting duplicated, consider doing this.